charts/ocis/docs/values-desc-table.adoc

```diff
@@ -1086,6 +1098,12 @@ a| [subs=-attributes]
 a| [subs=-attributes]
 `false`
 | Disables SSL certificate checking for connections to the oCIS http apis. Not recommended for production installations.
+| insecure.ocmInsecure
```

Comment:
should it also be used in OCM_OCM_PROVIDER_AUTHORIZER_VERIFY_REQUEST_HOSTNAME ?

Reply:
I wonder why it defaults to false for the oCIS product. This might violate the security by default principle!?

Reply:
Can't answer those questions. @micbar @butonic ?

Reply:
OCM_OCM_PROVIDER_AUTHORIZER_VERIFY_REQUEST_HOSTNAME is not about ssl certificate validation. OCM uses a service registry and sometimes the request might be routed to a different host. Nevertheless, we should default to secure. PR in https://github.com/owncloud/ocis/pull/10307

Reply:
So there is no need to configure this setting in the k8s world?

Reply:
It is about verifying host names in a mesh network of ocm instances. Not oll of them might run in the same k8s ;-) We should expose this in the config.

Reply:
addressed in https://github.com/owncloud/ocis-charts/pull/701/commits/0f012212cca7c4a8f8f1579d1315505eafc7cda9